lib/byebug/helpers/toggle.rb

```diff
@@ -26,14 +24,22 @@ def enable_disable_breakpoints(is_enable, args)
           selected_breakpoints = all_breakpoints.select do |b|
             selected_ids.include?(b.id)
           end
+          return selected_breakpoints
```

Comment:
You took more time replying to my comments than it would have taken you to address them. For example, my first request was to remove a single line of code. Instead, you wrote wrote a comment telling me to do it myself. Certainly not the best way to get changes in.

Anyways, I'll do this myself!